cpp/src/parquet/encryption/read_configurations_test.cc

```diff
@@ -103,9 +103,9 @@ class TestDecryptionConfiguration
   // This vector will hold various decryption configurations.
   std::vector<std::shared_ptr<parquet::FileDecryptionProperties>>
       vector_of_decryption_configurations_;
-  std::string kFooterEncryptionKey_ = std::string(kFooterEncryptionKey);
-  std::string kColumnEncryptionKey1_ = std::string(kColumnEncryptionKey1);
-  std::string kColumnEncryptionKey2_ = std::string(kColumnEncryptionKey2);
+  SecureString kFooterEncryptionKey_ = kFooterEncryptionKey;
```

Comment:
`const` seems warranted here
```suggestion
  const SecureString kFooterEncryptionKey_ = kFooterEncryptionKey;
```

Reply:
Doesn't that hold for all `k...` members in this class (`TestDecryptionConfiguration`) and `TestEncryptionConfiguration` as well? Is it worth it given this are tests?